Documentation/ceph-cluster-crd.md

```diff
@@ -340,6 +340,9 @@ Based on the configuration, the operator will do the following:
 
 \* Internal cluster traffic includes OSD heartbeats, data replication, and data recovery
 
+Only OSD pods will have Public as well as Cluster networks attached. Rest of the Ceph component pods and CSI pods will have Only Public network attached.
+Rook Ceph Operator will not have any networks attached.
```

Comment:
Does this mean the operator is not expected to need any networks attached? Then how does the operator connect to the cluster to run ceph commands? I'm just not clear how all the networks are connected.